cleanlab/multilabel_classification/dataset.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+import numpy as np
+from cleanlab.filter import _find_multilabel_issues_per_class
+from cleanlab.internal.multilabel_utils import get_onehot_num_classes
+from collections import defaultdict
+
+
+def common_multilabel_issues(
+    labels=list,
+    pred_probs=None,
+    *,
+    class_names=None,
+    confident_joint=None,
+) -> pd.DataFrame:
+    """Summarizes which classes in a multi-label dataset appear most often mislabeled overall.
+
+    This method works by providing any one (and only one) of the following inputs:
+
+    1. ``labels`` and ``pred_probs``, or
+    2. ``confident_joint``
+
+    Only provide **exactly one of the above input options**, do not provide a combination.
+
+    Parameters
+    ----------
+    labels : List[List[int]]
+        Refer to documentation for this argument in :py:func:`filter._find_multilabel_issues_per_class <cleanlab.filter._find_multilabel_issues_per_class>` for further details.
+
+    pred_probs : np.ndarray, optional
+      Refer to documentation for this argument in :py:func:`filter._find_multilabel_issues_per_class <cleanlab.filter._find_multilabel_issues_per_class>` for further details.
+
+
+    class_names : Iterable[str]
+        A list or other iterable of the string class names. The list should be in the order that
+        matches the label indices. So if class 0 is 'dog' and class 1 is 'cat', then
+        ``class_names = ['dog', 'cat']``.
```

Comment:
Add sentence:

If provided, these names will be used for ...

Reply:
https://github.com/cleanlab/cleanlab/blob/3c1524496ec519d772cc322da637b80badfe764a/cleanlab/multilabel_classification/dataset.py#L52